pages/about/translating/resources/dubbed-videos.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "How to Provide a Dubbed Version of a WAI Video"
+nav_title: Dubbed Videos
+lang: en
+last_updated: 2025-11-05
+description: Help make the Web accessible to people with disabilities around the world. We appreciate your contributions to translating W3C WAI accessibility resources.
+
+permalink: /about/translating/resources/dubbed-videos/
+ref: /about/translating/resources/dubbed-videos/
+
+github:
+  label: wai-translation-guidance
+
+image: /content-images/about/social-translations.png
+feedbackmail: wai@w3.org
+
+footer: |
+  <p><strong>Editors:</strong> <a href="https://www.w3.org/People/Shawn/">Shawn Lawton Henry</a> and Rémi Bétin.</p>
+---
+
+{::nomarkdown}
+{% include box.html type="start" h="2" title="Summary" class="full" %}
+{:/}
+
+This page guides you through the steps to provide a dubbed version of a video on the WAI website.
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{::options toc_levels="2" /}
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Page Contents" class="simple" %}
+{:/}
+
+-   TOC is created automatically.
+{:toc}
+
+{::nomarkdown}
+{% include toc.html type="end" %}
+{:/}
+
+## Overview
+
+There are 4 steps to provide a dubbed version of a WAI video:
+
+1. [Request permission](#permission)
+2. [Provide subtitles, or use existing ones](#subtitles)
+3. [Dub the video](#dubbing)
+4. [Deliver the final video to W3C](#review)
+
+## Step 1: Request permission {#permission}
+
+Before starting the production of a dubbed video, you must ask and receive permission from W3C. Please email your request to <group-wai-translations@w3.org>, or open a new issue [in the `wai-translations` GitHub repository  {% include_cached external.html %}](https://github.com/w3c/wai-translations/).
+
+In your message, state that you agree to contribute under the [Translation agreement](https://www.w3.org/WAI/about/translating/#translation-agreement). 
```

Comment:
that includes "Crediting" information which we will probably do differently for videos, since they are mostly likely done by and organization rather than an individual. Maybe put relevant info here instead of pointing them over there?